### DISCUSSION TITLE
Welcome to aptv Discussions!

### DISCUSSION THREAD

**Discussion by Kimentanm** [Kimentanm/aptv#53] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by petezhang35** [comment]
为啥加tg被封禁？

**Reply by Kimentanm** [reply]
因为入群有入群验证，如果5分钟内不验证会被自动封禁

**Reply by xufury** [reply]
> 因为入群有入群验证，如果5分钟内不验证会被自动封禁

请问被封禁怎么解封？还有验证码哪里获取？
